science/gfsview/Portfile

```diff
@@ -0,0 +1,46 @@
+#-*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
```

Comment:
The portfile's whitespace doesn't conform to this modeline; tabs should be changed to spaces.

Reply:
This should now be fixed.